src/Tree.hpp

```diff
@@ -135,6 +140,30 @@ namespace RRT
 
 
         /**
+         * A coefficient that determines how much we want to expand our stepsize when adaptive stepsize control.
+         */
+        int ascLimit() const {
+            return _ascLimit;
+        }
+        void setASCLimit(int lim) {
+            _ascLimit = lim;
+        }
+
+
+        /**
+         * Whether or not the tree is to run with adaptive stepsize control.
+         */
+        bool isDynamic() const {
+            return _isDynamic;
+        }
+        void setASC(int checked) {
```

Comment:
Ahh ok gotcha.  One thing to keep in mind is that although we're using the RRT code inside of a Qt-based app so we can test it, we want the RRT to be totally independent of which gui library we're using (if any at all).  It's fine that the UI has a three-state value for `checked`, but we should convert it to a `bool` before sending it to the model layer.  Outside the context of a Qt checkbox, having an int for whether or not asc is a little confusing.